### PR TITLE
Services: test fixes and code cleanup

### PR DESCRIPTION
**Pull request by martinpitt**
Some preparation, split out from #18422

### CODE REVIEW

File: pkg/systemd/services/service-details.jsx [hunk 1]
```diff
@@ -403,7 +404,7 @@ export class ServiceDetails extends React.Component {
 
             // Reload after unit/timer removal
             return Promise.all(deletions).then(() =>
-                systemd_client[this.props.owner].call(SD_OBJ, SD_MANAGER, "Reload", null)
+                systemd_client[this.props.owner].call(s_bus.O_MANAGER, s_bus.I_MANAGER, "Reload", null)
```
Comment (cockpituous) [inline pkg/systemd/services/service-details.jsx:407]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18445-20230303-111339-53caf4c9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
*grumble* nothing is ever so simple that you can't screw it up.. :exploding_head: Fixed.

**Comment by martinpitt** [comment]
I found another independent simplification. Not sure if this fixes anything, but I want to test it in isolation.

**Comment by martinpitt** [comment]
The /pybridge scenarios are really hard to get green with the 3x affected retries. They don't yet count into the statistics, and need the fixes from #18422.

**Comment by martinpitt** [comment]
I figured out the pesky page switching bug in #18443. It works well with the reworked services page, testing it here separately.

**Review by mvollmer — APPROVED** [review]
Looks all good to me. Thanks!

**Pull request merged**